houston/ardu/sandbox.py

```diff
@@ -213,86 +221,189 @@ def run(self, commands: Sequence[Command]) -> 'MissionOutcome':
             logger.debug("Mission uploaded")
             vcmds.wait_ready()
 
-            mylock = threading.Lock()
-            event = threading.Event()
+            wp_lock = threading.Lock()
+            wp_event = threading.Event()
```

Comment:
A little bit of explanation about how this event and lock are used would be helpful for debugging.